# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../glibc/glibc-2.13-do-not-link-nss_test1.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Origin: Linuxfromscratch Project

test-installation.pl script in that it tries to link a test program to a
library that isn't installed by make install

LFS is using the following sed command to fix it:

sed -i -e 's/"db1"/& \&\& $name ne "nss_test1"/' scripts/test-installation.pl

--- glibc-2.13/scripts/test-installation.pl.orig	2011-08-22 09:41:35.640080447 +0200
+++ glibc-2.13/scripts/test-installation.pl	2011-08-22 09:42:02.828085678 +0200
@@ -105,7 +105,7 @@
     # - libdb1 since it conflicts with libdb
     # - libnss1_* from glibc-compat add-on
     # - libthread_db since it contains unresolved references
-    if ($name ne "nss_ldap" && $name ne "db1"
+    if ($name ne "nss_ldap" && $name ne "db1" && $name ne "nss_test1"
 	&& !($name =~/^nss1_/) && $name ne "thread_db") {
       $link_libs .= " -l$name";
       $versions{$name} = $version;
